### ISSUE TITLE
v1.0.0-beta.9 build failed due to missing cmake dependency

### ISSUE THREAD

**Issue by yawnbox** [ZcashFoundation/zebra#4367]
## Description

Another build failure.

### Steps to Reproduce

I tried this:

```
cargo install --locked --git https://github.com/ZcashFoundation/zebra --tag v1.0.0-beta.9 zebrad
```

### Actual Behaviour

Instead, this happened: 

### Zebra Logs

```
error: failed to run custom build command for `prost-build v0.10.1`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installkxd3Nt/release/build/prost-build-4645591a0c4cc2d8/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=/home/christopher/.cargo/registry/src/github.com-1ecc6299db9ec823/prost-build-0.10.1/third-party/protobuf/cmake
  CMAKE_TOOLCHAIN_FILE_x86_64-unknown-linux-gnu = None
  CMAKE_TOOLCHAIN_FILE_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-unknown-linux-gnu = None
  CMAKE_GENERATOR_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_x86_64-unknown-linux-gnu = None
  CMAKE_PREFIX_PATH_x86_64_unknown_linux_gnu = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_x86_64-unknown-linux-gnu = None
  CMAKE_x86_64_unknown_linux_gnu = None
  HOST_CMAKE = None
  CMAKE = None
  running: "cmake" "/home/christopher/.cargo/registry/src/github.com-1ecc6299db9ec823/prost-build-0.10.1/third-party/protobuf/cmake" "-DCMAKE_INSTALL_PREFIX=/tmp/cargo-installkxd3Nt/release/build/prost-build-753a99b22a747d7b/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Debug"

  --- stderr
  thread 'main' panicked at '
  failed to execute command: No such file or directory (os error 2)
  is `cmake` not installed?

  build script failed, must exit now', /home/christopher/.cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.48/src/lib.rs:975:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `zebrad v1.0.0-beta.9 (https://github.com/ZcashFoundation/zebra?tag=v1.0.0-beta.9#c190e302)`, intermediate artifacts can be found at `/tmp/cargo-installkxd3Nt`

Caused by:
  build failed
```

### Operating System

Ubuntu 20.04 LTS x64

**Comment by teor2345** [comment]
Thanks, please install `protoc` (or `cmake`) and the build should succeed.

To close this issue, the Zebra developers could either:
- add  `protoc` (or `cmake`) as a dependency in the README,
- make `prost` into a test dependency, or
- just commit the files `prost` generates (it's only used for gRPC in the tests)

**Comment by nighthawk24** [comment]
Installing `cmake` via homebrew fixed the build issue!

**Comment by teor2345** [comment]
This should be fixed in the next beta release by PR #4369.

Until then, please install one of these packages as a workaround:
- `protobuf-compiler` (sometimes called `protobuf` or `protoc`)
- `cmake`

**Comment by yawnbox** [comment]
> 

Thank you, `sudo apt install cmake` allowed the build to work.
